catalog/app/app.tsx

```diff
@@ -99,12 +99,12 @@ const intercomUserSelector = (state) => {
   )
 }
 
-const sentryUserSelector = (state) => {
+const sentryUserSelector = (state: any) => {
   const { user: u } = Auth.selectors.domain(state)
   return u ? { username: u.current_user, email: u.email } : {}
 }
 
-const render = (messages) => {
+const render = (messages: any) => {
```

Comment:
My suggestion is `Record<'en', Record<string, string>>`, but I'm not quite sure

Reply:
yep, looks right. i've added typings for i18n module